tests/CMakeLists.txt

```diff
@@ -140,6 +140,8 @@ set_property(TEST cluster_launcher_if PROPERTY LABELS nonparallelizable_tests)
 
 add_test(NAME transition_to_if COMMAND tests/transition_to_if.py -v ${UNSHARE} WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
 set_property(TEST transition_to_if PROPERTY LABELS nonparallelizable_tests)
+add_test(NAME transition_to_if_lr COMMAND tests/transition_to_if.py -v -p 20 ${UNSHARE} WORKING_DIRECTORY ${CMAKE_BINARY_DIR})
+set_property(TEST transition_to_if_lr PROPERTY LABELS long_running_tests)
```

Comment:
Should it be `long_running_tests` while `transition_to_if` is `nonparallelizable_tests`?

Reply:
The 21 producers means it takes longer to transition; figured the run time now is enough to be a `long_running_tests`.